README.md

```diff
@@ -218,9 +287,80 @@ res.render("login", {
       size,
       color,
     }),
-  card: Card({
-    title: "Login With Ory",
-    children: Message({ message: "Woah there", severity: "info" }),
+  card: UserAuthCard({
+    title: !(flow.refresh || flow.requested_aal === "aal2")
+      ? "Sign In"
+      : "Two-Factor Authentication",
+    flow: flow as SelfServiceFlow,
+    flowType: "login",
+    cardImage: "ory-logo.svg",
+    additionalProps: {
+      forgotPasswordURL: "recovery",
+      signupURL: initRegistrationUrl,
+      logoutURL: logoutUrl,
+    },
   }),
 })
 ```
+
+---
+
+### Component System
+
+Ory Elements solely rely on React components since they are easy to write and
```

Comment:
Heh, this is a weird one. 

- "Ory Elements solely relies on..." (as a whole product)
- "Ory Elements solely rely on..." (as their individual parts)

I stumbled upon this when reading, because I am so used to elements being our complete suite of components. But in the bigger picture they really are more of a suite of individual elements, so the second version makes sense. Weird one! :)

Reply:
Actually below we are saying "Ory Elements is a component library" which implies they're a library instead of individual parts. So we should be consistent here and use the first case, IMO:

```suggestion
Ory Elements solely relies on React components since they are easy to write and
```